_service-details/secchamps.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Managed Security Champions Programme
+subtitle: Helping you build and improve this
+layout: service
+hero_height: is-small
+backlink: /services
+backname: Value
+---
+
+The industry is pretty unanimous that your organization needs a security champions programme. But where are you going to find time to develop a high value and high impact programme for this, alongside your already full schedule?
+
+That's where Bounce Security comes in.
+
+We use our significant experience in application security as well as our long experience of working directly with developers and within developement organizations to help you build and operate a security champions programme that works for your organization.
+
+We have seen enough Security Champion Programmes in the past to understand well the practices that makes a programme successful and anti-patterns that may reduce the programme's efficiency or even cause it to fail.

```

Comment:
I like this very much, but a long run-on sentence might have friction in parsing. Can we break it up to bit-size pieces?
And same with previous paragraph.

(also some spell check, like "developement")

Reply:
Split up the sentences and ran spellcheck on the whole thing